src/NzbDrone.Core/Indexers/IndexerBase.cs

```diff
@@ -17,6 +19,8 @@ namespace NzbDrone.Core.Indexers
     public abstract class IndexerBase<TSettings> : IIndexer
         where TSettings : IIndexerSettings, new()
     {
+        private static readonly Regex MultiRegex = new (@"\b(?<multi>multi)\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
As-is this won't match `_MULTI_` which some sites/release groups will use.
```suggestion
        private static readonly Regex MultiRegex = new (@"[_. ](?<multi>multi)[_. ]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```